lib/helpers/raw.js

```diff
@@ -1,9 +1,29 @@
 var path           = require('path')
 var fs             = require('fs')
+var _              = require('lodash')
 var TerraformError = exports.TerraformError = require("../error").TerraformError
 
 
 /**
+ * This is where we use harp.json to get some configs
+ * These are then merged against defaults
+ */
+try {
+  var harpConfig = exports.harpConfig = require(path.join(process.cwd(),'./harp.json'));
+} catch(e) {
+  try {
+    var harpConfig = exports.harpConfig = require(path.join(process.cwd(),'./_harp.json'));
+  } catch(e) {
+    var harpConfig = exports.harpConfig = {};
+  }
+}
+
+harpConfig = _.assign({
+  debug: true,
+  minify: false
+}, harpConfig)
+
+/**
```

Comment:
I'm not too familiar with lodash - what's the difference between the above and

```
harpConfig = Object.assign({
    debug: true,
    minify: false
}, harpConfig)
```

?